packages/polytechcolloscopeclient/lib/src/consts.dart

```diff
@@ -0,0 +1,37 @@
+enum Year {
+  first,
+  second,
+}
+
+class Consts {
+  static const String userAgent =
+      "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:96.0) Gecko/20100101 Firefox/96.0";
+
+  // HTTPS parce que le https est pas géré
+  static const String _base = "http://math.univ-lyon1.fr/colles/";
+
+  static const kholleURL = {
+    Year.first: "$_base?page=colles_1A",
+    Year.second: "$_base?page=colles_2A"
+  };
+
+  static const khollesStudentURL = {
+    Year.first: "$_base?page=colles_1A&id_etudiant=:id",
+    Year.second: "$_base?page=colles_2A&id_etudiant=:id"
+  };
+
+  static const Map monthsTranslation = {
```

Comment:
Using dart [extension](https://dart.dev/language/extension-methods) would be much cleaner, I think.

Reply:
I know, just wanted some quick tests during dev, i will do cleaner tests soon

Reply:
True, didnt know about extensions